core/includes/install.core.inc

```diff
@@ -547,25 +546,25 @@ function install_tasks($install_state) {
   // to the installation profile.
   $tasks = array(
     'install_select_language' => array(
-      'display_name' => st('Choose language'),
+      'display_name' => st('Choose your language'),
```

Comment:
I'm not sure about including the word "your" here. That makes it sound like it's the language of the user doing the installing, not necessarily the language the site itself will use. Can we revert this change and save it for a separate issue?